src/forms/registration-popup/index.jsx

```diff
@@ -86,7 +85,11 @@ const RegistrationForm = () => {
   const backendValidations = useSelector(getBackendValidations);
   const submitState = useSelector(state => state.register.submitState);
 
-  const autoSubmitRegForm = currentProvider && thirdPartyAuthApiStatus === COMPLETE_STATE && !isLoginSSOIntent;
+  const autoSubmitRegForm = (currentProvider
+      && thirdPartyAuthApiStatus === COMPLETE_STATE
+      && !isLoginSSOIntent
+      && queryParams?.authMode === 'Register'
```

Comment:
Will this not auto-register use when the popup opens if I am an unauthenticated user and I open home.edx.org?

Reply:
Yes, it will auto-register if the pipeline is active and user opens like this. 

I am wondering if this is real use case or whether is there any way to handle this?

Reply:
The only way I can think of is to append a query param to when a user goes through following flow
* User goes to register popup
* User clicks SSO
* Append query param
* If the query param exists, auto-register

This will help us identify if the pipeline was active and redirection is due to SSO flow or if user came from some other way.

I don't remember if we can add query param or not

Reply:
We can't add query params to the SSO pipeline.

Reply:
So its not achievable to set query param in SSO link

Reply:
I achieved it using local storage. Please review